test/hotspot/jtreg/compiler/intrinsics/string/TestStringIntrinsics2.java

```diff
@@ -43,6 +43,28 @@
  *        compiler.intrinsics.string.TestStringIntrinsics2
  */
 
+/*
+ * @test id=LargeHeap
+ * @bug 8254790
+ * @requires vm.bits == "64" & os.maxMemory > 5G
+ * @modules java.base/jdk.internal.misc
+ * @library /test/lib
+ *
+ * @build sun.hotspot.WhiteBox
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller sun.hotspot.WhiteBox
+ *
+ * @run main/othervm
```

Comment:
Is it useful to run all other tests in the TestStringIntrinsics2 using the large heap?